code/stack/stack.py

```
@@ -264,4 +268,364 @@ def test():
+        Q.enqueue(D.delete_first()) # queue = [4, 5, 3, 2, 1, 6, 7, 8] / deque -> []
+
+    for k in range(size):
+        
+        if len(Q) < 4:
+            D.add_last(Q.dequeue()) # deque = [..., 6, 7, 8]
+        else:
+            D.add_first(Q.dequeue()) # deque = [1, 2, 3, 5, 4]
+
+    return D
+
+def swap_indices_stack(D: Type[ArrayDeque], S: Type[ArrayStack], index_1: int, index_2: int) -> Type[ArrayDeque]:
```

Ruff (0.14.0)

Unused function argument: `index_1`

(ARG001)